files/usr/lib/jeos-firstboot

```diff
@@ -260,32 +260,34 @@ if [[ "$(ps h -o tty -p $$)" = tty[0-9]* ]]; then
 	fi
 fi
 
-# Find the location of the EULA
-# An EULA in /etc takes precedence
-EULA_FILE=/etc/YaST2/licenses/base/license.txt
-[ -e "${EULA_FILE}" ] || EULA_FILE=/usr/share/licenses/product/base/license.txt
-
-# Failsafe: If no license found, quit.
-if ! [ -e "$EULA_FILE" ]; then
-	d --msgbox $"No license found - cannot continue" 6 40
-	exit 1
-fi
+if [ -z "$JEOS_SKIP_EULA" ]; then
```

Comment:
`${}`

Reply:
Actually, it's more consistent to not use `{}` here, even if it would be better...

Reply:
Done

Reply:
I see, it is used in a non-very consistent way. Another thing to fix in the upper case for variables that are not influenced by an external shell.

Reverting here.